README.md

```diff
@@ -81,3 +81,212 @@ The following options are specific to this plugin:
   include A/B testing or localization specific output. Note: if you
   invoke the callback more than once, you should change the output
   filename to avoid overwriting previous invocations' output.
+
+
+## Examples
```

Comment:
I can move it if you think that’s best, but there are some advantages to adding these to the main README.

1. More people will see them! Since GitHub shows READMEs right on the project page, it will be quite visible. And it’s harder to miss examples than it is to miss a link.

2. They’ll appear in more places. In particular, the [npm module page on npmjs.com](https://www.npmjs.com/package/broccoli-eyeglass) shows the README right there.

Thoughts?